remote-api/src/main/resources/alfresco/public-rest-context.xml

```diff
@@ -174,6 +174,7 @@
                 <entry key="org.springframework.http.InvalidMediaTypeException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_UNSUPPORTED_MEDIA_TYPE}" />
                 <entry key="org.alfresco.rest.framework.core.exceptions.ServiceUnavailableException" value="503" />
                 <entry key="org.alfresco.service.cmr.dictionary.InvalidTypeException" value="409" />
+                <entry key="org.alfresco.service.cmr.repository.ArchivedIOException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_PRECONDITION_FAILED}" />
```

Comment:
Minor: As discussed,  n addition to 412 status code will this return unique errorKey or ...ApiDefault for now. 

If latter, could / should we resolve & remap to a new org.alfresco.rest.framework.core.exceptions.ArchivedIOException (or similar).

Reply:
Thanks @montgolfiere I've updated so that we now ultimately throw a `org.alfresco.rest.framework.core.exceptions.ArchivedContentException` after catching the `ArchivedIOException`, which extends `ApiException`. This allows us to control the errorKey in the response. 
To provide greater control I have also provided additional constructors within `ApiException` to allow us to have different errorKey and briefSummary.
The response will now look like the [comment below](https://github.com/Alfresco/alfresco-community-repo/pull/802#issuecomment-973171371), with errorKey being a default string provided by `ArchivedContentException` and the breifSummary being provided by the thrown `ArchivedIOException`.